src/components/ChatLog.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const chatEntryComponents = props.entries.map((entry) => {
```

Comment:
you can use index with map. This should get rid of your key on each child error.
```javascript
const chatEntryComponents = prop.entries.map((entry, i) => 
...
<ChatEntry
     key={i}